addons/web/static/tests/views/list_view_tests.js

```diff
@@ -7135,11 +7135,10 @@ QUnit.module("Views", (hooks) => {
             2,
             "the carret of closed groups should be right"
         );
-        assert.strictEqual(
-            value1Group.querySelector("span").style["padding-left"],
-            "2px",
-            "groups of level 1 should have a 2px padding-left"
-        );
+        const getCssVar = (groupEl, varName) => {
+            return getComputedStyle(groupEl.querySelector("span")).getPropertyValue(varName);
+        };
```

Comment:
Slight simplification:
```suggestion
        const getCssVar = (groupEl, varName) => getComputedStyle(groupEl.querySelector("span")).getPropertyValue(varName);
```